### PR TITLE
DOCS-6777 root access needed

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by sundippatel — APPROVED** [review]
LGTM

File: antora.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: runtime-fabric
```
Comment (kbostrom) [inline antora.yml:1]: INSTRUCTIONS FOR EDIT
Can you review this one line change (same fix for different versions) in

Comment (kbostrom) [reply antora.yml:1]: Went in to other file with the same change to identify which one to review here: root priv.

File: modules/ROOT/pages/install-manual.adoc [hunk 2]
```diff
@@ -146,6 +146,9 @@ mkdir -p /opt/anypoint/runtimefabric && cp ./rtf-install-scripts/scripts/init.sh
 
 . Run the `init.sh` script in privileged mode on the controller VM.
 +
+[NOTE]
+You must have root privileges to run the `init.sh` script.
++
```
Comment (kbostrom) [inline modules/ROOT/pages/install-manual.adoc:151]: Would it be useful to move the note info into the step: "As a user with root privileges, run the ..."?

**Review by kbostrom — APPROVED** [review]
Here you go!
I looked only at the line you specified, even though it looks like there is other new info?
Did you want me to look at that as well?

**Pull request closed without merging**